settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+        <InputListName>ClustersV</InputListName>
+        <TrackListName>TrackClustersV</TrackListName>
+        <ShowerListName>ShowerClustersV</ShowerListName>
+    </algorithm>
+    <algorithm type = "LArDLTrackShowerStreamSelection">
+        <InputListName>ClustersW</InputListName>
+        <TrackListName>TrackClustersW</TrackListName>
+        <ShowerListName>ShowerClustersW</ShowerListName>
+    </algorithm>
+
+    <!-- ThreeDTrackAlgorithms -->
+    <algorithm type = "LArThreeDTransverseTracks">
+        <InputClusterListNameU>TrackClustersU</InputClusterListNameU>
+        <InputClusterListNameV>TrackClustersV</InputClusterListNameV>
+        <InputClusterListNameW>TrackClustersW</InputClusterListNameW>
+        <OutputPfoListName>TrackParticles3D</OutputPfoListName>
+        <TrackTools>
+            <tool type = "LArClearTracks"/>
+            <tool type = "LArLongTracks"/>
+            <tool type = "LArOvershootTracks"><SplitMode>true</SplitMode></tool>
+            <tool type = "LArUndershootTracks"><SplitMode>true</SplitMode></tool>
+            <tool type = "LArOvershootTracks"><SplitMode>false</SplitMode></tool>
+            <tool type = "LArUndershootTracks"><SplitMode>false</SplitMode></tool>
+            <tool type = "LArMissingTrackSegment"/>
+            <tool type = "LArTrackSplitting"><MaxAbsoluteShortDeltaX>6.0</MaxAbsoluteShortDeltaX></tool>
+            <tool type = "LArLongTracks"><MinMatchedFraction>0.75</MinMatchedFraction><MinXOverlapFraction>0.75</MinXOverlapFraction></tool>
+            <tool type = "LArTracksCrossingGaps"><MinMatchedFraction>0.75</MinMatchedFraction><MinXOverlapFraction>0.75</MinXOverlapFraction></tool>
+            <tool type = "LArMissingTrack"/>
+        </TrackTools>
+    </algorithm>
+    <algorithm type = "LArThreeDLongitudinalTracks">
+        <InputClusterListNameU>TrackClustersU</InputClusterListNameU>
+        <InputClusterListNameV>TrackClustersV</InputClusterListNameV>
+        <InputClusterListNameW>TrackClustersW</InputClusterListNameW>
+        <OutputPfoListName>TrackParticles3D</OutputPfoListName>
+        <TrackTools>
+            <tool type = "LArClearLongitudinalTracks"/>
+            <tool type = "LArMatchedEndPoints"/>
+        </TrackTools>
+    </algorithm>
+
+    <algorithm type = "LArThreeDTrackFragments">
+        <MinClusterLength>5.</MinClusterLength>
+        <InputClusterListNameU>TrackClustersU</InputClusterListNameU>
+        <InputClusterListNameV>TrackClustersV</InputClusterListNameV>
+        <InputClusterListNameW>TrackClustersW</InputClusterListNameW>
+        <OutputPfoListName>TrackParticles3D</OutputPfoListName>
+        <TrackTools>
+            <tool type = "LArClearTrackFragments"/>
+        </TrackTools>
+        <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+    </algorithm>
+
+    <!-- ThreeDShowerAlgorithms -->
+    <algorithm type = "LArDLPfoCharacterisation">
```

Comment:
**There's an open question as to whether we still want these algs** - would be things for which we'd built something that is flagged as a shower out of clusters from the three track streams.